test/e2e/federated-service.go

```diff
@@ -34,6 +35,9 @@ const (
 
 	FederatedServiceName    = "federated-service"
 	FederatedServicePodName = "federated-service-test-pod"
+
+	KubeDNSConfigMapName      = "kube-dns"
```

Comment:
Yeah, but this file isn't the appropriate place for this variable. If this is the name of the constant that you want to reuse everywhere, it should probably in a more generic or appropriate location. Why is it in federated service test?